File: src/main/java/sc/iview/ui/ContextPopUp.java
package sc.iview.ui;

import graphics.scenery.Node;

import javax.swing.*;

public class ContextPopUp extends JPopupMenu {

    public ContextPopUp(Node n) {
        add( new JMenuItem("Name: " + n.getName()) );
        add( new JMenuItem("Type: " + n.getNodeType()) );
    }

}

File: src/main/java/sc/iview/ui/ContextPopUp.kt
package sc.iview.ui

import graphics.scenery.Node
import javax.swing.JMenuItem
import javax.swing.JPopupMenu

class ContextPopUp(n: Node) : JPopupMenu() {
    init {
        add(JMenuItem("Name: " + n.name))
        add(JMenuItem("Type: " + n.nodeType))
    }
}